src/enums/filingNames.ts

```diff
@@ -2,4 +2,9 @@ export enum FilingNames {
   INCORPORATION_APPLICATION = 'Incorporation Application',
   REGISTRATION = 'Registration',
   VOLUNTARY_DISSOLUTION = 'Voluntary Dissolution',
+  DISSOLUTION_FIRM = 'Dissolution',
+}
+
+export enum FilingTypesSubTitle {
+  SOLE_PROP_SUB_TITLE = 'Confirm the following information, select the dissolution date and certify your dissolution before filing.',
```

Comment:
I'm OK with declaring a new, related enum in the same file.

However, this message seems specific to a dissolution, and I don't get that from its key. I don't see where this is used so I can't form an opinion on how this string should be stored.

Also, I think this line is too long. Check the linting results.

Reply:
OK, I saw your new getter.

I left a comment on your getter. I think this needs a different solution, so I'm resolving this conversation and will look at your new solution.